fluent.syntax/docs/api/parser.rst

```diff
@@ -0,0 +1,7 @@
+Parsing
```

Comment:
The file is called `parser` which shows up in the URL. Should that be `parsing.rst` instead? Or, would it make sense to just use the name of the modules verbatim for the titles of the pages?

Reply:
I'm torn on this. I chose the title of the page to tell readers what we're doing, not so much about what that code is.

Re URL and page names, I have another idea, and that is to move `api/index.rst` to `api.rst` or `reference.rst`, and move `parser.rst` and `serializer.rst` to the top-level, too. But to keep their include structure.

Then URLs would be `fluent.syntax/parser.html`, which is almost the module name. I toyed around calling it `fluent.syntax/api/fluent.syntax.parser.html`, but that looks weird, even w/out the `api` in there.

What do you think?

Reply:
> Re URL and page names, I have another idea, and that is to move api/index.rst to api.rst or reference.rst, and move parser.rst and serializer.rst to the top-level, too. But to keep their include structure.

I like it. Since they'd be top-level, would it make sense to use the gerung forms in the file names then? We'd have `fluent.syntax/reference.html`, `fluent.syntax/parsing.html`, and  `fluent.syntax/serializing.html`.